testdata/src/main/java/com/hedvig/app/testdata/dashboard/Data.kt

```diff
@@ -0,0 +1,9 @@
+package com.hedvig.app.testdata.dashboard
+
+import com.hedvig.android.owldroid.graphql.InsuranceQuery
+import com.hedvig.app.testdata.dashboard.builders.InsuranceDataBuilder
+import java.time.LocalDate
+
+val DASHBOARD_DATA_WITH_RENEWAL_DATE = InsuranceDataBuilder(
+    InsuranceQuery.UpcomingRenewal(renewalDate = LocalDate.now(),
```

Comment:
You can make these parameters be the default in the builder, with the motivation that we never change them.

Should we need to change them, we can re-introduce them at that point.